neuronav/envs/grid_env.py

```diff
@@ -58,7 +58,7 @@ def __init__(
             raise Exception("No valid OrientationType provided.")
         self.state_size *= self.orient_size
         self.agent_pos = [0, 0]
-        self.reward_locs = {}
+        self.objects = {} # a dict with keys of "reward" and "markers", each a dict
```

Comment:
Can you structure this as `self.objects = {'markers':{}, 'rewards':{}} ?

Reply:
done